<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HUY-PC" ignored="0" name="TestSuite1" tests="10" failures="3" timestamp="26 thg 11 2018 14:35:52 GMT" time="90.576" errors="0">
  <testcase name="TC01" time="9.399" classname="Railway.LoginTest"/>
  <testcase name="TC02" time="4.256" classname="Railway.LoginTest"/>
  <testcase name="TC03" time="1.126" classname="Railway.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [There was a problem with your login and/or errors exist in your form.] but found [Invalid username or password. Please try again.]">
      <![CDATA[java.lang.AssertionError: expected [There was a problem with your login and/or errors exist in your form.] but found [Invalid username or password. Please try again.]
at Railway.LoginTest.TC03(LoginTest.java:41)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TC03 -->
  <testcase name="TC04" time="2.095" classname="Railway.LoginTest"/>
  <testcase name="TC05" time="5.474" classname="Railway.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [You have used 4 out of 5 login attempts. After all 5 have been used, you will be unable to login for 15 minutes.] but found [Invalid username or password. Please try again.]">
      <![CDATA[java.lang.AssertionError: expected [You have used 4 out of 5 login attempts. After all 5 have been used, you will be unable to login for 15 minutes.] but found [Invalid username or password. Please try again.]
at Railway.LoginTest.TC05(LoginTest.java:53)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TC05 -->
  <testcase name="TC06" time="14.818" classname="Railway.LoginTest"/>
  <testcase name="TC07" time="3.43" classname="Railway.LoginTest"/>
  <testcase name="TC01" time="14.706" classname="Railway.RegisterTest"/>
  <testcase name="TC02" time="1.971" classname="Railway.RegisterTest"/>
  <testcase name="TC01" time="4.894" classname="Railway.ChangePasswordTest">
    <failure type="java.lang.AssertionError" message="Your password has been updated must be displayed expected [Your password has been updated] but found [Your password has been updated!]">
      <![CDATA[java.lang.AssertionError: Your password has been updated must be displayed expected [Your password has been updated] but found [Your password has been updated!]
at Railway.ChangePasswordTest.TC01(ChangePasswordTest.java:21)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- TC01 -->
</testsuite> <!-- TestSuite1 -->
